docs/Makefile

```diff
@@ -18,26 +18,34 @@ ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees  $(SPHINXOPTS) .
 
 # This runs if you run make -C docs from the root dir
 # This has to go above help to work by default
-all: html
+all: linkcheck
 
 help:
 	@echo "Please use \`make <target>' where <target> is one of"
 	@echo "  all        generate the complete documentation api and regular"
 	@echo "  html       make only the HTML files from the existing rst sources"
 	@echo "  clean      clean all local doc directories out"
 	@echo "  serve      serve the docs locally using sphinx autobuild"
+	@echo "  linkcheck  run sphinx link check on the docs and produce output w broken links"
 	
 # Run: make -C docs clean 
 clean:
 	rm -rf $(BUILDDIR)/html/*
 	rm -rf $(BUILDDIR)/doctrees
+	rm -rf $(BUILDDIR)/linkcheck
 	rm -rf .ipynb_checkpoints
 
 html: 
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
+linkcheck: html
+	$(SPHINXBUILD) -b linkcheck $(ALLSPHINXOPTS) $(BUILDDIR)/linkcheck
```

Comment:
link check now will produce an output file output.txt with all broken things ... much easier to read as well vs looking at the CLI output of sphinx/linkcheck